pkg/machine/qemu/machine.go

```diff
@@ -1085,9 +1093,6 @@ func (v *MachineVM) startHostNetworking() (string, apiForwardingState, error) {
 	}
 
 	attr := new(os.ProcAttr)
-	// Pass on stdin, stdout, stderr
-	files := []*os.File{os.Stdin, os.Stdout, os.Stderr}
-	attr.Files = files
```

Comment:
Will this attach any fds to stdin/out/err for gvproxy?  I am not sure why you set /dev/null for qemu but not here for gvproxy?

Reply:
the reason behind this, right or wrong, was because for we do actually pass in an fd for socket use.  To keep the order of the fd the same, i passed in /dev/nulls.  And then with gvproxy, I just removed the fds because we really don't need them.  Would you rather I went through the exercise of not passing in /dev/nulls and fix the fd ordering for the socket?

Reply:
The problem is you cannot just not set fds for the child process. If there are no fds and the child starts opening fds then they will get placed on 0,1 and 2 which causes all kinds of problems. For example if gvproxy tries to write something to stdout/err now it will write to a random fd and this will likely fail or cause other side effects.